packages/common-tests/browser-tests/page-model/eventSearchPage.ts

```diff
@@ -3,11 +3,16 @@ import { t } from 'testcafe';
 import { initTestI18n as i18n } from '../../../common-i18n/src';
 import type { AppNamespace } from '../types/app-namespace';
 
+type SearchProps = {
+  searchText: string;
+};
+
+const defaultSearchProps: SearchProps = { searchText: 'helsinki' };
+
 class EventSearchPage {
-  searchText = 'helsinki';
   private appNamespace: 'appHobbies' | 'appEvents' | 'appSports';
 
-  constructor(appNamespace: AppNamespace) {
+  constructor(appNamespace: AppNamespace | undefined = 'appEvents') {
```

Comment:
I can recheck this, but the thing was that I had a common util that took advantage of this method. I can try to refactor it too or just give "any" app there in that code. I thought that it would be less confusing if the class constructor says that it's ok to use it without the appNamespace too.

Reply:
Okay, I'll change this so that the constructor allows only 1 of the 3 namespaces (as a mandatory) and I just use any of them in the common code.